employees/views.py

```diff
@@ -83,39 +91,45 @@ def make_year_list_for_filtering_in_emp_detail(employee):
     else:
         return False
 
-def position_list_and_filtering(self, queryset):
-    positions = {
-        'position_sale': 'Sale',
-        'position_production': 'Production',
-        'position_other': 'Other',
-        'position_none': 'None',
-    }
-    exclude_list = []
-    for key in positions.keys():
-        if self.request.GET.get(key) is None:
-            exclude_list.append(positions[key])
-            queryset = queryset.exclude(position=positions[key])
-    queryset = queryset.exclude(position=None)
-    return queryset
+
+def former_and_current_filtering(self, queryset):
+    # queryset = queryset.exclude(currently_employed=None) don't need to take caro of None, column Not Null in Data Base
+    if self.request.is_ajax():
+        if self.request.GET.get('current_employees'):
+            queryset = queryset.exclude(currently_employed=False)
+        if not self.request.GET.get('current_employees'):
```

Comment:
This should be replaced with `else` statement since there are only two options.

Reply:
fixed :)